yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/HumanProfileAdapter.java

```diff
@@ -103,11 +105,15 @@ public Date getDateOfBirth() {
 
     @Override
     public Boolean isAgeVerified() {
-        Boolean isAgeOver = wrapped.findAttributeStartingWith(ATTRIBUTE_AGE_OVER, Boolean.class);
-        Boolean isAgeUnder = wrapped.findAttributeStartingWith(ATTRIBUTE_AGE_UNDER, Boolean.class);
+        Boolean isAgeOver = parseFromStringValue(wrapped.findAttributeStartingWith(ATTRIBUTE_AGE_OVER, String.class));
+        Boolean isAgeUnder = parseFromStringValue(wrapped.findAttributeStartingWith(ATTRIBUTE_AGE_UNDER, String.class));
         return isAgeOver != null ? isAgeOver : isAgeUnder;
     }
 
+    private Boolean parseFromStringValue(String value) {
+        return value == null ? null : parseBoolean(value);
```

Comment:
does this make more sense --> `return value == null ? value : parseBoolean(value);`

Reply:
Can do